tests/go/go-kafka-serde/client.go

```diff
@@ -87,14 +92,15 @@ func main() {
 	log.Debugf("Protocol: %v", *protocol)
 	log.Debugf("Count: %v", *count)
 	log.Debugf("Security Settings: %v", securitySettings)
+	log.Debugf("Timeout: %v", *timeout)
 
 	prot, ok := ParseProtocol(*protocol)
 
 	if !ok {
 		util.Die("Failed to parse protocol %v", *protocol)
 	}
 
-	tc, err := CreateTestClient(brokers, topic, sr_addr, consumer_group, prot, *count, securitySettings)
+	tc, err := CreateTestClient(brokers, topic, sr_addr, consumer_group, prot, *count, time.Duration(*timeout), securitySettings)
```

Comment:
Should we multiple this by seconds here so downstream usage doesn't need to understand that it should be seconds?